code/game/machinery/doors/firedoor.dm

```diff
@@ -36,6 +36,14 @@
 	if(!density)
 		return ..()
 	return 0
+	
+/obj/machinery/door/firedoor/ex_act(severity)
+	switch(severity)
+		if(1.0)

```

Comment:
It's a remnant of one of the original dmb-extracted ss13 builds, iirc. Their decompiler recognized all of the numbers as floats (which is true in BYOND).